go.mod

```diff
@@ -4,16 +4,15 @@ go 1.18
 
 require (
 	github.com/docker/distribution v2.8.2+incompatible
-	k8s.io/api v0.28.0
-	k8s.io/apimachinery v0.28.0
+	k8s.io/api v0.27.2
+	k8s.io/apimachinery v0.27.2
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	sigs.k8s.io/controller-runtime v0.15.1
```

Comment:
`sigs.k8s.io/controller-runtime` also need to be compatible with kubernetes 1.27?

Reply:
yes, controller-runtime v0.15.1 is compatible with api v0.27.2